### ISSUE TITLE
Allow vendor prefixed properties to come before others when sorting alphabetically

### ISSUE THREAD

**Issue by viveleroi** [hudochenkov/stylelint-order#161]
Our app has the following block:

```
body {
    -moz-osx-font-smoothing: grayscale;
    -webkit-font-smoothing: antialiased;
    color: #363E47;
}
```

This plugin is warning that the vendor-prefixed properties are out of order, but this is the order we want. I don't see any way to fix this as-is.

**Comment by hudochenkov** [comment]
Prefixes kind of ignored to determine an order. Both `-moz-osx-font-smoothing` and `-webkit-font-smoothing` are treated as `font-smoothing`. It makes clear that `font-smoothing` should be after `color` alphabetically.

**Comment by viveleroi** [comment]
So there's no way to force the order we desire (prefixed first, then non-prefixed properties)?
